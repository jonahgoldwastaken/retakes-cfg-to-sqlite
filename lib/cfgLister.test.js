const cfgLister = require('./cfgLister')

test('cfgLister to list all files in cfgs folder', () => {
  expect(cfgLister('../cfgs')).toMatchSnapshot('cfgFolderContents')
})
