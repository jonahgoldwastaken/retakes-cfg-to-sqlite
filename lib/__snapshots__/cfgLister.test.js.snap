// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cfgLister to list all files in cfgs folder: cfgFolderContents 1`] = `
Array [
  "de_aztec.cfg",
  "de_biome.cfg",
  "de_cache.cfg",
  "de_cbble.cfg",
  "de_contra_b3.cfg",
  "de_crown.cfg",
  "de_dust.cfg",
  "de_dust2.cfg",
  "de_inferno.cfg",
  "de_mill_ce.cfg",
  "de_mirage.cfg",
  "de_nuke.cfg",
  "de_overpass.cfg",
  "de_santorini.cfg",
  "de_season.cfg",
  "de_season_rc1.cfg",
  "de_subzero.cfg",
  "de_toscan.cfg",
  "de_train.cfg",
  "de_tuscan.cfg",
  "de_vertigo.cfg",
]
`;
